Maddori.Apple/Maddori.Apple/Screen/Home/Home/UIComponent/ChangeTeamView.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  ChangeTeamView.swift
+//  Maddori.Apple
+//
+//  Created by 이성호 on 2023/02/06.
+//
+
+import UIKit
+
+import SnapKit
+
+final class ChangeTeamView: UIView {
+    
+    // MARK: - FIXME: 데이터 더미 입니다.
+    let teamDataDummy: [String] = ["맛쟁이 사과처럼", "명예 소방관"]
+    
+    // MARK: - property
+    
+    private let titleLabel: UILabel = {
+        let label = UILabel()
+        label.font = .label2
+        label.text = TextLiteral.changeTeamViewLabel
+        label.textColor = .black100
+        return label
+    }()
+    private let collectionViewFlowLayout: UICollectionViewFlowLayout = {
+        let flowLayout = UICollectionViewFlowLayout()
+        flowLayout.scrollDirection = .vertical
+        flowLayout.sectionInset = UIEdgeInsets(top: 12, left: 0 , bottom: 16, right: 0)
+        flowLayout.itemSize = CGSize(width: UIScreen.main.bounds.width - SizeLiteral.leadingTrailingPadding*2, height: 59)
+        flowLayout.minimumLineSpacing = 8
+        return flowLayout
+    }()
+    lazy var teamCollectionView: UICollectionView = {
+        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: collectionViewFlowLayout)
+        collectionView.delegate = self
+        collectionView.dataSource = self
+        collectionView.register(TeamChangeCollectionViewCell.self, forCellWithReuseIdentifier: TeamChangeCollectionViewCell.className)
+        collectionView.backgroundColor = .white200
+        return collectionView
+    }()
+    private lazy var emptyView = EmptyTeamView()
```

Comment:
p2;

```suggestion
    private let emptyView = EmptyTeamView()
```

요거 let으로 해도 괜찮지않나 싶습니다!

Reply:
lazy 로 한 이유가 아마 api 연결에서 연결 후에 데이터가 있는지 확인 후에 있음 생성하는게 더 좋아보여서 미리 한건데 현재 PR에서는 지우는게 좋아보이는군요
[f4f1ca9](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/319/commits/f4f1ca9c5afe820b6581992c2211b5efe77e8aff)